invokeai/frontend/web/src/features/controlLayers/konva/CanvasEntity/CanvasEntityAdapterRasterLayer.ts

```diff
@@ -74,4 +80,79 @@ export class CanvasEntityAdapterRasterLayer extends CanvasEntityAdapterBase<
     const keysToOmit: (keyof CanvasRasterLayerState)[] = ['name', 'isLocked'];
     return omit(this.state, keysToOmit);
   };
+
+  private syncAdjustmentsFilter = () => {
+    const a = this.state.adjustments;
+    const apply = !!a && a.enabled;
+    // The filter operates on the renderer's object group; we can set filters at the group level via renderer
+    const group = this.renderer.konva.objectGroup;
+    if (apply) {
+      const filters = group.filters() ?? [];
+      let nextFilters = filters.filter((f: unknown) => f !== AdjustmentsSimpleFilter && f !== AdjustmentsCurvesFilter);
```

Comment:
- `group.filters()` always returns `Filter[]`
- `f` should already be inferred as type `Filter`